### PR TITLE
Promisify and support Node.js >=4

### PR DESCRIPTION
**Pull request by kevva**
Depends on https://github.com/sindresorhus/default-browser-id/pull/1/ getting published.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,8 @@
 sudo: false
 language: objective-c
+before_install:
+  - brew update
+  - brew unlink node
 install: brew install node
 script:
   - npm install
```
Comment (sindresorhus) [inline .travis.yml]: Same as the other PR

Comment (kevva) [inline .travis.yml]: Done!

File: .editorconfig [hunk 2]
```diff
@@ -10,6 +10,3 @@ insert_final_newline = true
 [package.json]
```
Comment (sindresorhus) [inline .editorconfig]:
```
[{package.json,*.yml}]
```

File: package.json [hunk 3]
```diff
@@ -38,12 +38,16 @@
     "applescript"
   ],
   "dependencies": {
-    "bundle-name": "^1.0.0",
-    "default-browser-id": "^1.0.0",
+    "bundle-name": "^2.0.1",
+    "default-browser-id": "^12.0.0",
```
Comment (sindresorhus) [inline package.json]: One too many `1` here ;)

Comment (kevva) [inline package.json]: :black_joker:

**Comment by sindresorhus** [comment]
tumblr_mk77lvyp4t1qdlh1io1_250

**Comment by SamVerschueren** [comment]
Hahaha nice!

**Pull request merged**